andela/iceman-backend
# chore setup sequelize 167735021

**What does this PR do?**

  This PR setup sequelize for the project

  **Description of Task to be completed?**

 - Installed sequelize, pg and pg-hstore
 - Enable migrations
- Created .sequelizerc file to manage sequelize-cli configurations settings
- Convert the generated config.json file to config.js  to enabled the use of environmental variable in the  migration config folder
- Convert the index.js in the models folder to use latest JavaScript standard
- Created User model to check if everything is working fine.


  **How should this be manually tested?**

  - Firstly, please make sure my first pull request is merge that is setting babel and env, because it requires babel to transpile some module and  environmental variables  to work
  - Secondly follow the Sequelize Setup instruction attached to the README

  **Any background context you want to provide?**

  -  I created test:dbscript in the scripts section of package.json to ease the process of creating the necessary models (tables) in the test database during testing.

  **What are the relevant stories?**

  #167735021 - Setup Sequelize - https://www.pivotaltracker.com/story/show/167735021
